### ISSUE TITLE
Session Table

### ISSUE THREAD

**Issue by BradleyLamitie** [HarshaKota/MSCS-710L-Project#10]
Should we create the Sessions table to store start and end timestamps?

**Comment by HarshaKota** [comment]
Yes, I totally forgot about that. Will create and test the table now. Expect the commit tonight.

**Comment by BradleyLamitie** [comment]
Okay thanks, I kind of forgot about it until now. I'm reviewing the code now, and trying to see if I can resolve any bugs we currently have.
